.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  pull_request:
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - name: linux (with gcc)
+            os: ubuntu-latest
+
+          - name: MacOS (with clang)
+            os: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: bootstrap
+      run: bootstrap.sh
```

Comment:
Probably this should be
```
./bootstrap.sh
```
See also avalentino/erfa@03717dfe84d81fc036cfe4d106ab4b55949e6fd9